config/setup/xutil.ads

```diff
@@ -6,7 +6,7 @@
 --                                                                          --
 --                                 S p e c                                  --
 --                                                                          --
---          Copyright (C) 1992-2012, Free Software Foundation, Inc.         --
+--          Copyright (C) 1992-2018, Free Software Foundation, Inc.         --
```

Comment:
Why? There is no change in this file.

Reply:
I just copied it from gnat sources.

Reply:
Ok,  but there is no change here so the update is wrong to me. Some projects do update the copyright notice each start of year, AWS does not.